Khan/react-render-server
# Make sure we use the PORT env var that GAE provides for listening

I noticed while testing my logging changes that GAE really wants us to use the `PORT` environment variable but we were hard-coded to use `8080`. This changes that.

## Test Plan

1. Deploy this commit SHA
2. Fire a test payload at it from `curl` and check all works
3. Update webapp with this commit SHA
4. Check that development server works too
